Assignment6/RearrangingCars.py

```diff
@@ -0,0 +1,71 @@
+class Parking:
+
+    def __init__(self, cars):
+        self._cars = cars
+        self._size = len(cars)
+
+    def cars(self):
+        return self._cars.copy()
+
+    def size(self):
+        return self._size
+
+    def rearrange(self, result):
+        """
+        Rearrange cars in self._cars to get the result and print out each step.
+        :param result: list(int)
+        :return: None or raise ValueError
+        """

```

Comment:
Cool! PyCharm seems to be nice XD. I was thinking of adding comments near the type, perhaps something like this:
```python
"""
:param result: list(int) - a list representing the current order of the cars
"""
What do you think? 
```